scripts/docs.js

```diff
@@ -0,0 +1,38 @@
+const fs = require('fs');
+const path = require('path');
+
+const API_PATH = path.resolve(__dirname, '../website/docs/api');
+const SIDEBARS_FILE = path.resolve(__dirname, '../website/sidebars.js');
+const sidebarsData = require('../website/sidebars');
+
+['index.md', 'globals.md'].forEach((fileName) => {
+  if (fs.existsSync(`${API_PATH}/${fileName}`)) {
+    fs.unlinkSync(`${API_PATH}/${fileName}`);
+  }
+});
+
+if (sidebarsData.guides && sidebarsData.guides.Interfaces) {
+  const interfaces = sidebarsData.guides.Interfaces;
+  delete sidebarsData.guides.Interfaces;
+
+  if (sidebarsData.api) {
+    sidebarsData.api.Interfaces = interfaces;
```

Comment:
all looks good. Could you just change this to `sidebarsData.api.Widgets = interfaces`?
We want the heading to be "Widgets"